applications/dashboard/settings/structure.php

```diff
@@ -28,6 +28,7 @@
    ->PrimaryKey('RoleID')
    ->Column('Name', 'varchar(100)')
    ->Column('Description', 'varchar(500)', TRUE)
+   ->Column('Type', array('Guest', 'Unconfirmed', 'Applicant', 'Member', 'Moderator', 'Administrator'), TRUE)
```

Comment:
These enums also need to be constants on the `PermissionModel`. Use the constants in code.